dev/io.openliberty.netty.internal.impl/bnd.bnd

```diff
@@ -28,11 +28,6 @@ Import-Package: \
   io.openliberty.netty.internal.exception, \
   ${defaultPackageImport}
 
-# reuse the existing translated messages from the TCP Channel
```

Comment:
Is it safe to get rid of this, since the messages are still in use?

Reply:
There gone.